mindsdb/utilities/security.py

```diff
@@ -33,22 +33,65 @@ def clear_filename(filename: str) -> str:
     return filename
 
 
-def validate_urls(urls, allowed_urls):
+def _split_url(url: str) -> tuple[str, str]:
```

Comment:
I would name it 'get_url_origin'
Also it can return f'{scheme}://{netloc}', the behavior will be the same